styles/index.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap');
```

Comment:
Would be much better when you import links for fonts in html page rather than CSS

Reply:
- [ ] Kindly add a why  to the requested change, you can also add link to a resources the student can go to for better understanding 👍🏿